src/js/popup.js

```diff
@@ -541,21 +637,25 @@ $(document).ready(function () {
         }
     });
 
-    // When the edit template has been compiled
+    // When the sandbox compiles a template
     $(window).on('message', function (event) {
         event = event.originalEvent;
-        if (event.data.html) {
-            $('#templateEditor').append(event.data.html);
+        if (event.data.content === 'template-editor') {
+            if (event.data.html) {
+                $('#templateEditor').append(event.data.html);
 
-            $('textarea').each(function (index) {
-                if ($(this).val()) {
-                    $(this).trigger('autoresize');
-                }
-            });
+                $('textarea').each(function (index) {
+                    if ($(this).val()) {
+                        $(this).trigger('autoresize');
+                    }
+                });
 
-            if (event.data.openTemplate) {
-                openCollapsible(event.data.openTemplate);
+                if (event.data.openTemplate) {
+                    openCollapsible(event.data.openTemplate);
+                }
             }
+        } else if (event.data.content === 'domain-list') {
+            $('.collection').append(event.data.html);
```

Comment:
Should probably check `if (event.data.html) {` like the other case

Reply:
:+1: